service/rpc/servicer.go

```diff
@@ -419,6 +419,320 @@ func (s *Servicer) BlockTransaction(ctx context.Context, request *types.BlockTra
 	}, nil
 }
 
+func (s *Servicer) Call(
+	ctx context.Context,
+	callRequest *types.CallRequest,
+) (*types.CallResponse, *types.Error, error) {
+	switch callRequest.Method {
+	case isAccount:
+		address, ok := callRequest.Parameters["address"].(common.Address)
+		if !ok {
+			err := errors.New("failed to parse 'address' as common.Address")
+			return nil, LogErrValidation(err), err
+		}
+
+		callOpts, err := buildCallOpsFromCallRequest(ctx, callRequest)
+		if err != nil {
+			return nil, LogErrValidation(err), err
+		}
+
+		registry, err := registry.New(s.cc)
+		account, err := registry.GetAccountsContract(ctx, nil)
+		if err != nil {
+			return nil, LogErrValidation(err), err
+		}
+
+		isAccount, err := account.IsAccount(callOpts, address)
+		if err != nil {
+			return nil, LogErrValidation(err), err
+		}
+
+		result := map[string]interface{}{
+			"isAccount": isAccount,
+		}
+
+		return &types.CallResponse{
+			Result:     result,
+			Idempotent: true,
+		}, nil, nil
+
+	case getVoteSigner:
+		address, ok := callRequest.Parameters["address"].(common.Address)
+		if !ok {
+			err := errors.New("failed to parse 'address' as common.Address")
+			return nil, LogErrValidation(err), err
+		}
+
+		callOpts, err := buildCallOpsFromCallRequest(ctx, callRequest)
+		if err != nil {
+			return nil, LogErrValidation(err), err
+		}
+
+		registry, err := registry.New(s.cc)
+		account, err := registry.GetAccountsContract(ctx, nil)
+		if err != nil {
+			return nil, LogErrValidation(err), err
+		}
```

Comment:
Would it make sense to abstract this shared code to get Accounts used in both isAccount / getVoteSigner into a separate function? This way, anything that touches that contract can be updated in one place if anything changes

If you think this makes sense, I imagine a similar utility can be made for anything touching a group